modules/govuk_jenkins/manifests/job/search_reindex_with_new_schema.pp

```diff
@@ -0,0 +1,27 @@
+# == Class: govuk_jenkins::job::search-reindex-with-new-schema
+#
+# Test rebuilding the search indexes and reindexing all content
+#
+class govuk_jenkins::job::search_reindex_with_new_schema (
+  $app_domain = hiera('app_domain'),
+) {
+
+  $check_name = 'search-reindex-with-new-schema'
+  $service_description = 'Rebuild new search indexes with up to date settings and mappings, and reindex all content from the existing indexes.'
+  $job_url = "https://deploy.${app_domain}/job/search-reindex-with-new-schema/"
+
+  file { '/etc/jenkins_jobs/jobs/search_reindex_with_new_schema.yaml':
+    ensure  => present,
+    content => template('govuk_jenkins/jobs/search_reindex_with_new_schema.yaml.erb'),
+    notify  => Exec['jenkins_jobs_update'],
+  }
+
+  @@icinga::passive_check { "${check_name}_${::hostname}":
+    service_description => $service_description,
+    host_name           => $::fqdn,
+    freshness_threshold => 619200,
```

Comment:
Should we configure this passive check to only run on integration? The job won't be run regularly on staging and production, which I think will trigger (incorrect) passive alerts.

Reply:
I blame @MatMoore which is super easy to do as he isn't here. I have updated this with a flag in the hieradata.